### ISSUE TITLE
Build error on VS2012

### ISSUE THREAD

**Issue by tobybear** [schellingb/ZillaLib#3]
At the moment, neither the actually library nor any Zillalib project compile on VS2012, the build process is immediately terminated with this cryptic error message:
>Error: 'pathToFile' cannot be an empty string ("") or start with the null character.

The reason for this is found in the project settings under "Configuration Properties - General - Build Log File". Here the invalid string value "nul:" is given, which makes VS fail.
If this is removed in all configurations for all solutions, it works fine again.

This setting can alternatively also be changed directly in the ZillaLib-vs.vcxproj file, just remove this line: 
`<ItemDefinitionGroup><BuildLog><Path>nul:</Path></BuildLog></ItemDefinitionGroup>`

**Comment by schellingb** [comment]
Hi there

Thank you so much for trying this project! Especially in VS2012 which I don't have installed anymore but I blindly assumed something that builds in VC6 and VS2013 would be fine.

Overwriting the BuildLog to "nul:" I wasn't happy about anyway but I just saw a way to get rid of another useless output file so I took it. I'll remove it or add a condition to ignore it in 2012.
Good you found this, the error message alone is indeed useless.

Thanks!
